synapse/tools/sync_020.py

```diff
@@ -104,6 +128,12 @@ class SyncMigratorApi(s_stormsvc.StormSvc, s_cell.CellApi):
     async def stopSync(self):
         return await self.cell.stopSync()
 
+    async def enableMigrationMode(self):
```

Comment:
Does this persist or would an admin need to re-disable migration mode on a restart?

Reply:
Whenever the sync is restarted it will re-enable migrationMode by default.  One scenario where it would have to be manually re-enabled is if the cortex is restarted, and the sync service is left up.  Another is the sync service is restarted and you want to run sync with migrationMode=False, since that's overriding the default.  It might be better to hold onto this option and set whenever the connection to the src is established/re-established.

Reply:
Updated to persist an override and only auto-toggle if not set